Invalid `main` in package.json

```
Error: Cannot find module '/Users/zachleat/Code/eleventy/node_modules/urlpattern-polyfill/dist/index.cjs'. Please verify that the package.json has a valid "main" entry
```

I don’t see an index.cjs in the dist folder locally on `rc1`—did I miss something?

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "https://github.com/kenchris/urlpattern-polyfill"
   },
-  "main": "./dist/index.cjs",
+  "main": "./dist/index.umd.js",
   "module": "./dist/index.js",
   "unpkg": "./dist/index.umd.js",
   "files": [

